packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -80,6 +133,72 @@ export class FormSchemaService {
     return formSchema;
   }
 
+  private getFormTranslationsByUuidFromServer(formUuid: string) {
+    const formTranslations: ReplaySubject<any> = new ReplaySubject(1);
```

Comment:
This looks like it duplicates a lot of code from `getFormSchemaWithReferences()`. Why do we need two functions doing the same things?